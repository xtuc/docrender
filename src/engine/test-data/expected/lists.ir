(
    content: [
        Clear,
        Clear,
        List([
            InlineText([
                Text("A"),
            ], true),
            InlineText([
                Text("B"),
            ], true),
            InlineText([
                Text("C"),
            ], true),
            InlineText([
                Text("D"),
            ], true),
            InlineText([
                Text("E"),
            ], true),
            InlineText([
                Text("F"),
            ], true),
        ]),
        Clear,
        Paragraph([]),
        Paragraph([]),
        Clear,
        List([
            InlineText([
                Text("A"),
            ], true),
            InlineText([
                Text("B"),
            ], true),
            InlineText([
                Text("C"),
            ], true),
            InlineText([
                Text("D"),
            ], true),
            InlineText([
                Text("E"),
            ], true),
            InlineText([
                Text("F"),
            ], true),
        ]),
        Clear,
        Paragraph([]),
        Paragraph([]),
        Clear,
        List([
            InlineText([
                Text("A"),
            ], true),
            InlineText([
                Text("B"),
            ], true),
            InlineText([
                Text("C"),
            ], true),
            InlineText([
                Text("D"),
            ], true),
            InlineText([
                Text("E"),
            ], true),
            InlineText([
                Text("F"),
            ], true),
        ]),
        Clear,
        Paragraph([]),
        Paragraph([]),
        Clear,
        List([
            InlineText([
                Text("A"),
            ], true),
            InlineText([
                Text("B"),
            ], true),
            InlineText([
                Text("C"),
            ], true),
            InlineText([
                Text("D"),
            ], true),
            InlineText([
                Text("E"),
            ], true),
            InlineText([
                Text("F"),
            ], true),
        ]),
        Clear,
        Paragraph([]),
        Paragraph([]),
        Clear,
        List([
            InlineText([
                Text("a "),
            ], true),
            InlineText([
                Text("b"),
            ], true),
            InlineText([
                Text("C"),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Learn more about:"),
            ], true),
        ]),
        Clear,
        Clear,
        List([
            InlineText([
                TextWithOptions("Integrate using Edge Side Includes (ESI)", (
                    link: Some("https://docs.publish-bot.com/edge-side-includes"),
                    styles: [
                        ("text-decoration", "underline"),
                        ("color", "rgb(6.666667%, 33.333336%, 80%)"),
                    ],
                )),
            ], false),
            InlineText([
                TextWithOptions("Display a Table of Contents", (
                    link: Some("https://docs.publish-bot.com/display-a-table-of-contents"),
                    styles: [
                        ("text-decoration", "underline"),
                        ("color", "rgb(6.666667%, 33.333336%, 80%)"),
                    ],
                )),
            ], false),
            InlineText([
                TextWithOptions("Direct URL", (
                    link: Some("https://docs.publish-bot.com/direct-url"),
                    styles: [
                        ("text-decoration", "underline"),
                        ("color", "rgb(6.666667%, 33.333336%, 80%)"),
                    ],
                )),
            ], false),
        ]),
    ],
    styles: {
        "pbNORMAL_TEXT": [
            ("line-height", "1.7249999"),
        ],
    },
)