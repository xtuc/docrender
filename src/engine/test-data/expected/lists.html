<style>
  .pbNORMAL_TEXT {
    line-height: 1.7249999;
  }
</style>
<ul>
  <li>A</li>
  <li>B</li>
  <li>C</li>
  <li>D</li>
  <li>E</li>
  <li>F</li>
</ul>
<ul>
  <li>A</li>
  <li>B</li>
  <li>C</li>
  <li>D</li>
  <li>E</li>
  <li>F</li>
</ul>
<ul>
  <li>A</li>
  <li>B</li>
  <li>C</li>
  <li>D</li>
  <li>E</li>
  <li>F</li>
</ul>
<ul>
  <li>A</li>
  <li>B</li>
  <li>C</li>
  <li>D</li>
  <li>E</li>
  <li>F</li>
</ul>
<ul>
  <li>a</li>
  <li>b</li>
  <li>C</li>
</ul>
<p>Learn more about:</p>
<ul>
  <li>
    <a
      style="text-decoration: underline; color: rgb(6.666667%, 33.333336%, 80%)"
      href="https://docs.publish-bot.com/edge-side-includes"
      >Integrate using Edge Side Includes (ESI)</a
    >
  </li>
  <li>
    <a
      style="text-decoration: underline; color: rgb(6.666667%, 33.333336%, 80%)"
      href="https://docs.publish-bot.com/display-a-table-of-contents"
      >Display a Table of Contents</a
    >
  </li>
  <li>
    <a
      style="text-decoration: underline; color: rgb(6.666667%, 33.333336%, 80%)"
      href="https://docs.publish-bot.com/direct-url"
      >Direct URL</a
    >
  </li>
</ul>
